### PR TITLE
feat: ✨ yarn pnp commands run with `node -r .pnp.js`

### LINKED ISSUE #141: Run yarn PnP commands faster
Yarn berry scripts can be run with
```
node -r ./.pnp.js path/to/$$virtual/package/binary.js
```

This halves the start time

I am already working on a PR for this

### PR DESCRIPTION
**Pull request by remorses**
fix #141

### CODE REVIEW

**Comment by remorses** [comment]
Time comparison in a large yarn pnp monorepo (cannot share link, it's from work)

Yarn
```
❯ time yarn tsc --version
Version 3.7.5
yarn tsc --version  3,64s user 0,39s system 145% cpu 2,775 total
```

Ultra
```
❯ time bin/ultra.js tsc --version
✔ $ tsc --version 514ms
  │ Version 3.7.5
  │ 
success ✨ Done in 1.273s
bin/ultra.js tsc --version  1,70s user 0,20s system 147% cpu 1,292 total
```

File: src/runner.ts [hunk 1]
```diff
@@ -162,13 +162,41 @@ export class Runner {
...
   }
 
+  // cached pnpFile location
+  private pnpFile: string | undefined
+
+  findPnpJsFile(cwd: string = process.cwd()): string | undefined {
+    if (this.pnpFile) {
+      return this.pnpFile
+    }
+    const dir = findUp(".pnp.js", cwd)
```
Comment (merceyz) [inline src/runner.ts:172]: Can be either `.pnp.js` or `.pnp.cjs`

**Review by folke — CHANGES_REQUESTED** [review]
Looks good to me except for the comment by @merceyz 

If you can resolve that one, I'll merge this and push a new release.

Thank you!

**Comment by remorses** [comment]
Added support for `.pnp.cjs`

**Review by folke — APPROVED** [review]

**Comment by folke** [comment]
Merged and released! 🚀

**Pull request merged**